release-notes/release-3.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: February 28, 2024
+---
+
+# Features
+- Metrics Explorer in Settings provides users a single location to view all of their collected metrics -  users can search for metrics by resource name, key, or label names
+- Dynamic filters in the alert module for users based on their existing alert configurations
+- Users can mute their already created Alerts in the details screen
+- Alerts are dynamically changed based on the last triggered alert for that rule
+- Data Ingestion Intervals allow users to control the interval on which they ingest data for any new dashboard
+- Kafka Integration
+
+# Changes
+- Increased trace ingestion capabilities (200,000 traces/min) to decrease probability of crashes
```

Comment:
Let's not put a number on this as it could accidentally look like not enough to some customers. I would rephrase as "Improvements to our trace ingestion backend allowing for large scale deployments"